.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: Release
+# This workflow is triggered when publishing a GitHub release
+# yamllint disable-line rule:truthy
```

Comment:
not sure you need this comment if you are not making use of validation tools such as pre-commit or any yaml linters (up to you)

Reply:
Ok I will remove it